src/features/runner.ts

```diff
@@ -9,29 +9,29 @@ import * as vscode from 'vscode';
 export class AnsiblePlaybookRunProvider {
     private disposableTerminal: vscode.Terminal | undefined;
     constructor(private vsCodeExtCtx: vscode.ExtensionContext) {
-        // this.disposableTerminal = this.makeTerminal();
```

Comment:
Urgh... Did I write this? Need to remember to remove in a separate PR.

Reply:
I doubt and I agree we need to take it out.